package.json

```diff
@@ -25,6 +25,7 @@
   },
   "homepage": "https://github.com/JuanManu59/protractor-workshop-2020#readme",
   "dependencies": {
+    "jasmine-spec-reporter": "^5.0.2",
```

Comment:
please install jasmine and protractor as dev-dependencies (`npm install --save-dev`)